<launch> 
 <!-- Docking -->
 <node pkg="base_nav" name="docking_node" type="docking_node" output="screen">
     <param name="goal_tolerance" value="0.05" />
     <param name="max_speed" value="0.2" />
     <param name="P_dist" value="3.0" />
     <param name="I_dist" value="1.5" />
     <param name="P_ang" value="3.0" />
     <param name="I_ang" value="1.5" />
     <param name="max_integral" value="10" />
     <param name="control_period" value="0.1" />
 </node>
</launch>
